core/src/main/java/pl/agh/edu/engine/bank/TransactionType.java

```diff
@@ -1,10 +1,33 @@
 package pl.agh.edu.engine.bank;
 
 import pl.agh.edu.serialization.KryoConfig;
+import pl.agh.edu.utils.LanguageString;
 
 public enum TransactionType {
-	INCOME,
-	EXPENSE;
+	RECEIVED_CREDIT(true, "transaction.type.receivedCredit"),
+	ROOM_RENTING_INCOME(true, "transaction.type.roomRentingIncome"),
```

Comment:
Zastanawiam się czy nie przydałby się podział na pokoje pod względem rozmiaru lub rangi. Skoro pokoje to główny income to dobrze byłoby mieć dokładniejszy wgląd